test/cargo_flags/rust/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "flags-lib"
+version = "0.1.0"
+edition = "2021"
```

```suggestion
edition = "2018"
```

This should also fix the CI failure